credentials/credentials.go

```diff
@@ -38,7 +43,16 @@ func getLicenseKeySecretId(conf *config.Configuration) string {
 		return conf.LicenseKeySecretId
 	}
 
-	return defaultSecretId
```

Comment:
@nc-wittj It doesn't seem to be documented behavior, but "NEW_RELIC_LICENSE_KEY" was previously returned as a fallback secret id. I'm trying to preserve that behavior for backwards compatibility.